CMakeLists.txt

```diff
@@ -10,6 +10,8 @@
 cmake_minimum_required( VERSION 3.12 )
 project( fms VERSION 2020.4.0 LANGUAGES C Fortran )
 
+option(OPENMP "Build FMS with OpenMP support" OFF)
```

Comment:
Wondering if we should make the default `ON` ...

Reply:
I think the default should be `ON` to match what is currently the default. I tested this on my Mac, I can turn off OpenMP and it builds w/o it.

Reply:
Made the change.